config/deploy.js

```diff
@@ -0,0 +1,22 @@
+const { raw } = require('config/raw');
+const { consoleLogger } = require('../src/logger');
+
+module.exports = {
+    server: {
+        corsEnabledFor: ['*'],
+        logger: raw(consoleLogger('error')),
+        port: 8080,
+        importerSendTxEndpoint: 'fake',
+    },
+    db: {
+        user: 'fake',
```

Comment:
is it possible to explicitly load the environment variables in this file? It would make the mapping clearer.

eg: `user: process.env.DB_USER || (throw "DB_USER missing")`

Reply:
I suppose that's possible.